Source/Core/DolphinQt/RenderWidget.cpp

```diff
@@ -530,11 +530,10 @@ void RenderWidget::PassEventToPresenter(const QEvent* event)
     const u32 key = static_cast<u32>(key_event->key() & 0x1FF);
 
     const char* chars = nullptr;
-    QByteArray utf8;
 
     if (is_down)
     {
-      utf8 = key_event->text().toUtf8();
+      QByteArray utf8 = key_event->text().toUtf8();
```

Comment:
Same problem here. `utf8` goes out of scope but `chars` keeps a pointer to it.

Reply:
I don't see what's wrong here either

Reply:
When exiting the `if (is_down)` statement, `utf8` will be destroyed and `chars` will become a dangling pointer pointing to destroyed data _(see `chars = utf8.constData()`)_.